p2p/conn/connection_test.go

```diff
@@ -116,6 +119,40 @@ func TestMConnectionStatus(t *testing.T) {
 	assert.Zero(status.Channels[0].SendQueueSize)
 }
 
+func TestMConnectionPongTimeoutResultsInError(t *testing.T) {
```

Comment:
Think we need more tests:

- ping/pong works correctly (ie. we don't disconnect)
- receiving many pongs is benign
- pong timeout results in error